README.md

```diff
@@ -3,11 +3,28 @@
 
 # Solrb
 
-Object-Oriented approach to Solr in Ruby. 
+Object-Oriented approach to Solr in Ruby.
 ## Installation
 
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'solr'
+gem 'solrb'
+```
+
+## Configuration
+
+```
+Solr.configure do |config|
+  config.url = http://yoursolrurl/
+
+  config.define_field_types do |t|
```

Comment:
here you don't define field types, you define fields, see how they're defined in the source:

https://github.com/machinio/parts-solr/blob/master/parts/conf/schema.xml#L43

So it should say `config.define_dynamic_fields` here